sigs.yaml

```diff
@@ -2363,34 +2363,36 @@ sigs:
     owners:
     - https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-tools/master/OWNERS
 workinggroups:
-- dir: wg-apply
-  name: Apply
+- dir: wg-api-expression
+  name: API Expression
   mission_statement: >
-    Improve the state of declarative object management by fixing `kubectl apply`,
-    moving the logic from the client to the server. This move also enables new/fixed
-    features included in the scope of this working-group: diff, dry-run and prune.
+    Enable API authors to better serve API consumers, by improving and documenting
+    all aspects of Kubernetes API development.
 
-    Resources can be found in [this Google drive folder](https://drive.google.com/drive/folders/1wlpgkS2gFZXdp4x2WlRsfUBxkFlt2Gx0)
+    [See full Mission Statement](https://docs.google.com/document/d/1XYbQXfge2qKM9psksfC5XZnW8hybtLqL1EcJLU4JwKg).
 
   stakeholder_sigs:
   - API Machinery
-  - CLI
-  label: apply
+  - Architecture
+  label: api-expression
   leadership:
     chairs:
-    - github: lavalamp
-      name: Daniel Smith
+    - github: apelisse
+      name: Antoine Pelisse
       company: Google
+    - github: kwiesmueller
+      name: Kevin Wiesmueller
+      company: //SEIBERT/MEDIA GmbH
   meetings:
   - description: Regular WG Meeting
     day: Tuesday
     time: "9:30"
     tz: PT (Pacific Time)
     frequency: biweekly
-    url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
+    url: https://calendar.google.com/calendar/embed?src=seibert-media.net_s7j7r6giqmr2l6g8ev5idh3ij8%40group.calendar.google.com&ctz=Europe%2FBerlin
```

Comment:
I don't know how we're managing community calendars any more.  Copying it to my own calendar is needed to get notifications, but seems fragile.  Is this going to be on a distinct sig-api-machinery calendar?

Sadly, this timeslot conflicts with a just-established sig-multi-cluster working session.  I'll just have to juggle.  Not enough EU friendly timeslots in a week.

Reply:
If you join the mailing list, you'll be automatically invited and should get notifications. That's how we had it for apply and I have the event in my personal calendar.

https://groups.google.com/g/kubernetes-wg-api-expression

Reply:
I did it this way, as I couldn't find a guideline on community calendars and invites, so a group and calendar is how I've seen it before.